manifest.json

```diff
@@ -46,6 +44,15 @@
     {
       "name": "Get_User_By_Identifier"
     },
+    {
+      "name": "template-criar"
+    },
+    {
+      "name": "template-editar"
+    },
+    {
+      "name": "send-message"
+    },
```

Comment:
These policies are written in Portuguese, are they right?

Reply:
Yes, you can see all available policies here: https://vtex.myvtex.com/admin/resources-list
Many have Portuguese names